phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/ConnectionInfo.java

```diff
@@ -114,6 +115,15 @@ public static ConnectionInfo create(String url, ReadOnlyProps props, Properties
         return create(url, conf, props, info);
     }
 
+    public static ConnectionInfo create(String url, ReadOnlyProps props,
+            Properties info, boolean isServerConnection)
+            throws SQLException {
+        Configuration conf = HBaseFactoryProvider.getConfigurationFactory().getConfiguration();
+        ConnectionInfo connInfo = create(url, conf, props, info);
+        connInfo.isServerConnection = isServerConnection;
```

Comment:
I think we should initialize `isServerConnection` the same way we are initializing principal, keytab, user and haGroup variables.

Reply:
All the other variables are being set via Builder. Lets not break that pattern for `isServerConnection`

Reply:
Done.